### PR TITLE
Fixes #18925: Simplify the ncf_tests scenario and remove the setup part in it

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/18925

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

File: rudder-tests/lib/scenario.py [hunk 1]
```diff
@@ -284,6 +287,7 @@ def run_testinfra(self, target, test, error_mode=Err.CONTINUE, **kwargs):
 
     tmp_report_file = self.workspace + "/tmp_report.json"
     pytest_cmd = ['--capture=no', '-v', '--test_data=%s'%json.dumps(input_data), '--token=%s'%self.token, '--webapp_url=%s'%webapp_url, testfile, "--json-report", "--json-report-file=" + tmp_report_file]
+    #pytest_cmd = ['--capture=no', '-v', '--test_data=%s'%json.dumps(input_data), '--token=%s'%self.token, '--webapp_url=%s'%webapp_url, testfile, "--json-report", "--json-report-file=" + tmp_report_file]
```
Comment (ncharles) [inline rudder-tests/lib/scenario.py:290]: why this line ?

**Comment by Fdall** [comment]
Commit modified

**Review by ncharles — APPROVED** [review]

**Pull request merged**